app/components/pages/Dashboard/index.js

```diff
@@ -20,6 +20,7 @@ const BoxHiddenFromMobile = styled(Box)`
 `
 
 const TabbedSubmissions = styled(Tabs)`
+  min-height: 300px;
   margin-bottom: ${th('space.2')};
```

Comment:
Which means that this just needs to be `space.3` @jyenes

Reply:
done